docs/rest-api/source/includes/wp-api-v3/order.md

```diff
@@ -0,0 +1,141 @@
+# Order
+
+The order API allows you to perform operations such as capture a terminal payment, create an intent, create a customer, etc. for a particular order.
+
+## Create a terminal intent
```

Comment:
Overall, I wasn't able to understand this section. 

- The request has `<order_id>` in it's path.
- There's a post param `payment_intent_id: string`
- example doesn't pass `payment_intent_id` 

I think that 
```
POST params
payment_intent_id: string
```

is mistakenly put in this section but it's missing from next one - "Capture a terminal payment"

Reply:
Indeed. It was earlier fixed in 1d20d47 but the deployed docs were not regenerated later. In fact, docs will only be regenerated when this PR is merged.